backend/src/core/mail/application/mail-service.ts

```diff
@@ -0,0 +1,38 @@
+import { createMailTransport } from '~/configuration/mail-transport';
+import { CONFIG } from '~/configuration/config';
+import { injectable } from 'inversify';
+import Email from 'email-templates';
+import { MailType } from '~/shared/enums/enums';
+import { MailProps } from '~/shared/types/types';
+
+@injectable()
+export class MailService {
+  async sendEmail(receiver: string, mailType: MailType, props: MailProps): Promise<void> {
+    const transport = await createMailTransport();
```

Comment:
Mail transport should be initialized one time during creating di container (see initialization PrismaClient in backend/src/secondary-adapters/postgres/postgres-container-module.ts)